cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocRuntimeException.java

```diff
@@ -40,7 +53,7 @@ public DataprocRuntimeException(@Nullable String operationId, Throwable cause) {
 
   @Override
   public String toString() {
-    return getMessage();
+    return String.format("ErrorTags: %s,  Msg: %s", errorTags.toString(), getMessage());
```

Comment:
I am wondering if message handling should be centralized at interface level. Also it should handle empty string

Reply:
Yes, it gets handled in LogAppender. This is just the toString for this exception if it gets explicitly called.

Reply:
Removed.